app/src/main/java/org/jellyfin/androidtv/startup/SelectUserFragment.java

```diff
@@ -49,13 +49,19 @@ protected void addAdditionalRows(ArrayObjectAdapter rowAdapter) {
         HeaderItem usersHeader = new HeaderItem(rowAdapter.size(), mApplication.getString(R.string.lbl_select_user));
         ItemRowAdapter usersAdapter = new ItemRowAdapter(mServer, new CardPresenter(), rowAdapter);
         usersAdapter.Retrieve();
```

Comment:
I do not know how to add a simple button to this `ItemRowAdapter`. Is it even possible?

Reply:
It looks like you found a method, right?

Reply:
No, when there are no users I create another adapter with `GridButtonPresenter`. When there are existing users, I don't know how to add this button since the adapter uses `CardPresenter`.

Reply:
I think you need to create your own adapter & presenter that doesn't use the current one. It gives more flexibility and allows for better customization.

Might be a bit of work though.